mvcc/backend/backend.go

```diff
@@ -150,11 +150,16 @@ type IgnoreKey struct {
 }
 
 func (b *backend) Hash(ignores map[IgnoreKey]struct{}) (uint32, error) {
-	h := crc32.New(crc32.MakeTable(crc32.Castagnoli))
-
 	b.mu.RLock()
 	defer b.mu.RUnlock()
-	err := b.db.View(func(tx *bolt.Tx) error {
+	return ComputeHash(b.db, ignores)
+}
+
+// ComputeHash computes Hash value from db.
+func ComputeHash(db *bolt.DB, ignores map[IgnoreKey]struct{}) (uint32, error) {
```

Comment:
does this need to be exported or can this always be done through backend.Hash()?